setup.py

```diff
@@ -70,7 +70,8 @@
                  'Programming Language :: Python :: 3.7',
                  'Programming Language :: Python :: 3.8',
                  'Programming Language :: Python :: 3.9',
-                 'Programming Language :: Python :: 3.10'
+                 'Programming Language :: Python :: 3.10',
+                 'Programming Language :: Python :: 3.11'
```

Comment:
+ 3.12

Reply:
would you like to merge this `3.11` PR and `3.12` in followup PR?